### DISCUSSION TITLE
[Projects Beta] In page subtask and issue update

### DISCUSSION THREAD

**Discussion by krsoninikhil** [community/community#8836] [Projects and Issues]
It would be very easy to manage subtasks if issue can be opened and edited in the projects page itself instead of opening in new tab, similar to how it works with draft issues on project page. I would have kept the issue in draft state itself but then the checklist updation doesn't work. 

Managing subtasks can be made easier.

**Comment by Abadiq313** [comment]
EMOJICODE

**Comment by nkolev92** [comment]
Likely a duplicate of https://github.com/github/feedback/discussions/5381.
